code_review_interview/frontend/PeopleList.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+
+import { PeopleListCard } from './PeopleListCard';
+
+type PeopleListProps = {
+  people: any;
+};
+
+export const PeopleList: React.FC<PeopleListProps> = (props) => {
+  useEffect(() => {
+    props.people.sort((a, b) => a.name.localeCompare(b.name));
+  }, [props.people]);
+
+  if (!props.people) {
+    return 'Loading...';
+  }
+
+  return (
+    <div>
+      {props.people.map((person) => (
+        <PeopleListCard key={person.name} person={person} />
```

Comment:
suggestion (blocking): This could lead to key collision if 2 people have the same name -- consider incorporating the person's ID as well